src/NCronJob/Registry/RuntimeJobRegistry.cs

```diff
@@ -192,10 +192,6 @@ public bool TryRegister(Action<IRuntimeJobBuilder> jobBuilder, [NotNullWhen(fals
                 jobQueueManager.SignalJobQueue(jobDefinition.JobFullName);
             }
 
-            foreach (var entry in jobRegistry.DynamicJobRegistrations)
-            {
-                jobQueueManager.SignalJobQueue(entry.JobDefinition.JobFullName);
```

Comment:
Line 192 already does the signaling job, as a dynamic job is also added as a plain `JobDefinition` by `JobRegistry.AddDynamicJob()`